Clew/Controllers/ViewController.swift

```diff
@@ -1732,6 +1775,9 @@ class ViewController: UIViewController, ARSCNViewDelegate, SRCountdownTimerDeleg
                 
                 // erase current keypoint node
                 keypointNode.removeFromParentNode()
+//                pathObj?.removeFromParentNode()
```

Comment:
Same question as above.  Is there a reason why we don't have to remove the path node from the scene?